cmd/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	config "github.com/openebs/zfs-localpv/pkg/config/v1alpha1"
+	service "github.com/openebs/zfs-localpv/pkg/service/v1alpha1"
+	"github.com/openebs/zfs-localpv/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	_ = flag.CommandLine.Parse([]string{})
+	var config = config.Default()
+
+	cmd := &cobra.Command{
+		Use:   "openebs-csi-driver",
+		Short: "openebs-csi-driver",
+		Run: func(cmd *cobra.Command, args []string) {
+			run(config)
+		},
+	}
+
+	cmd.Flags().AddGoFlagSet(flag.CommandLine)
+
+	cmd.PersistentFlags().StringVar(
+		&config.RestURL, "url", "", "REST URL that points to maya api server",
```

Comment:
Are we dependent on Maya API Server?
Can we avoid this completely?
What is making this dependent on API server?

Reply:
removed it.